### PR TITLE
added description of SESSION_LENGTH variable

### PR DESCRIPTION
**Pull request by mkaesz-wandb**
## Description

Added description of SESSION_LENGTH variable

## Ticket

No

## Checklist

Check if your PR fulfills the following requirements. Put an `X` in the boxes that apply.

- [X] Files I edited were previewed on my local development server with `yarn start`. My changes did not break the local preview.
- [X] Build (`yarn docusaurus build`) was run locally and successfully without errors or warnings.
- [X] I merged the latest changes from `main` into my feature branch before submitting this PR.

### CODE REVIEW

File: docs/guides/hosting/iam/advanced_env_vars.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ Choose any of the following environment variables for your instance depending on
 | Environment variable | Description |
 |----------------------|-------------|
 | DISABLE_SSO_PROVISIONING | Set this to `true` to disable user auto-provisioning in your W&B instance. |
+| SESSION_LENGTH | To configure the SSO login session expiry, set this environment variable to the desired number of hours. The default value is 720 hours (24 hours * 30 days). Note: the value should be specified without the "h" unit. |
```
Comment (abhinavg6) [inline docs/guides/hosting/iam/advanced_env_vars.md]:
```suggestion
| SESSION_LENGTH | If you would like to change the default user session expiry time, set this variable to the desired number of hours. E.g., set it as `24` to configure session expiry time of 24 hours which is equivalent to a day. The default value is 720 hours which is equivalent to 30 days. |
```

**Review by abhinavg6 — CHANGES_REQUESTED** [review]
Request to change it as suggested. Thanks for doing this.

**Comment by mkaesz-wandb** [comment]
@abhinavg6 thank you. accepted it.

**Review by abhinavg6 — APPROVED** [review]
LGTM thanks

**Review by ngrayluna — APPROVED** [review]

**Pull request merged**